src/styles/components/_pane.scss

```diff
@@ -60,6 +60,9 @@
     transform: translateX(100%);
     transition: all 0.3s;
 
+    display: flex;
```

Comment:
@goutham-subramanyam _a Let’s revert these changes for Pane as it has some side effects.